docs/samples/Microsoft.ML.Samples/Dynamic/ImageClassification/ResnetV2101TransferLearningEarlyStopping.cs

```diff
@@ -57,7 +57,7 @@ public static void Example()
 
                 IDataView trainDataset = trainTestData.TrainSet;
                 IDataView testDataset = trainTestData.TestSet;
-
+                /*
```

Comment:
why are we commenting this out?

Reply:
I had to remove it but commented it out. Will remove it. The code snippet following it includes my API changes and hers.